manifests/apache/vhost.pp

```diff
@@ -35,7 +35,7 @@
   String[1] $user                              = $puppetboard::user,
   String[1] $group                             = $puppetboard::group,
   Stdlib::AbsolutePath $basedir                = $puppetboard::basedir,
-  String[1] $override                          = $puppetboard::override,
+  Array[String[1]] $override                   = $puppetboard::override,
```

Comment:
we could do something like `Variant[Array[String[1]],String[1]]` and later call `[$puppetboard::override].flatten`. that would keep it compatible with the older apache module releases. then it's not a breaking change on our side. If we do a breaking change we need to update our metadata.json as well (but the apache changes arent released yet)

Reply:
Yeah… I adjusted the code accordingly.